modules/administration-guide/partials/proc_configuring-bitbucket-servers.adoc

```diff
@@ -1,14 +1,73 @@
 // configuring-bitbucket-servers
-
+ 
 [id="configuring_bitbucket_servers_{context}"]
 = Configuring Bitbucket servers
+ 
+You can use the Bitbucket (BB) server as a project source supplier in {prod-short}}. To do so, configure the related CheCluster Custom Resource (CR) and {prod-short} instance devfile.
+
+The following chapter describes configuring BB for use with public repositories, where the use of additional certificates, such as TLS, may be required. This includes the following processes:
```

Comment:
I would better say `configuring {prod} for use with BB public repositories` (generally one cannot configure BB until he is not it's administrator)